dist/models/index.ts

```diff
@@ -62,6 +62,12 @@ interface generateBaseRequest {
    * text.
    */
   return_likelihoods?: "GENERATION" | "ALL" | "NONE";
+
+  /** Used to prevent the model from generating unwanted tokens or to incentivize it to include desired tokens
```

Comment:
this comment doens't really describe which way does way (I.e. negative = ?), also, are we continuing to update the node SDK?

Reply:
also mention that it is only for default models for now and min/max range of values?
https://github.com/cohere-ai/blobheart/blob/8dc5ca678518a4ecc8eb97a27a807c65f794d45c/docs/docs/generate.mdx#L198